module/rdpClientCon.c

```diff
@@ -2162,13 +2162,65 @@ rdpClientConSendPaintRectShmEx(rdpPtr dev, rdpClientCon *clientCon,
 }
 
 /******************************************************************************/
+static int
+rdpCapRect(rdpClientCon *clientCon, BoxPtr cap_rect, struct image_data *id)
```

Comment:
Would it be possible to add comments to new functions? The name is too short to be clear.